test/e2e/operator_test.go

```diff
@@ -237,8 +237,18 @@ func TestClusterOperatorStatusRelatedObjects(t *testing.T) {
 			Group:    "gateway.networking.k8s.io",
 			Resource: "gatewayclasses",
 		})
-		// This test runs before TestGatewayAPI, so we do *not* expect
-		// to see subscriptions, istios, or gateways in relatedObjects.
+		if gatewayAPIControllerEnabled, err := isFeatureGateEnabled(features.FeatureGateGatewayAPIController); err != nil {
+			t.Fatalf("Failed to look up %q featuregate: %v", features.FeatureGateGatewayAPIController, err)
+		} else if gatewayAPIControllerEnabled {
+			expected = append(expected, configv1.ObjectReference{
+				Group:     "gateway.networking.k8s.io",
+				Resource:  "gateways",
+				Namespace: "openshift-ingress",
```

Comment:
Should the namespace be blank here?

Reply:
Possibly.  My thinking was that we were telling people to create gateways in the "openshift-ingress" namespace, so the platform-managed ones should be there, we're allowing gateways in other namespaces as well (we just won't manage DNS for them).  Based on that, I suppose it makes sense to leave the namespace blank.